packages/font-glyphs/src/common/shapes.ptl

```diff
@@ -530,6 +494,36 @@ glyph-block CommonShapes : begin
 		local args : object [yRef y] sw swTerminal isTail noSwash [overshoot o]
 		return : WithKnotProxy [hookProxy args] : new FunctionInterpolator hookEndBlender args
 
+	glyph-block-export flatside
+	define flatside : namespace
+		export : define [ld l d u ada adb _af] : return
```

Comment:
If we are adding `flatside` to build the sides for circle-likes, then maybe we could perform overshoot processing inside these functions, as the function names already contain the information about which side the stroke is. An exmaple could be:
```
...
export define flex-params [ld]: begin
    local-parameter : x # X-position of the stroke, without overshoot
    local-parameter : d # Y-position of the ring's bottom, without overshoot
    local-parameter : u # Y-position of the ring's top, without overshoot
    local-parameter : af -- nothing # Additional function for the control knots
    local-parameter : ada -- ArchDepthA
    local-parameter : adb -- ArchDepthB
    local-parameter : overshoot -- OX


    if (u - d > ada + adb)
        list
            flat (x + overshoot) (u - ada) af
            curl (x + overshoot) (d + adb)
        g4 (x + overshoot) [mix d u (adb / (ada + adb))] af
```